code/stable/gamephysics/SRS/GamePhysics_SRS.tex

```diff
@@ -549,9 +556,7 @@ \subsubsection{Data Definitions}
               \item{${m_{T}}$ is the total mass of the rigid body (kg)}
               \end{symbDescription}
 \\ \midrule \\
-Notes & \hyperref[assumpOT]{A: objectTy}
-        
-        \hyperref[assumpOD]{A: objectDimension}
+Notes & All bodies are assumed to be rigid (from \hyperref[assumpOT]{A: objectTy}) and two-dimensional (from \hyperref[assumpOD]{A: objectDimension}).
```

Comment:
We can remove the assumption that the body is 2D.  The equation for centre of mass is the same for 2D as it is for 3D.  I just verified that the original case study version makes this same mistake.  Now that I noticed it though, we should remove it.  😄